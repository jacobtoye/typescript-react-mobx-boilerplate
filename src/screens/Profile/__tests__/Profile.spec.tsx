import * as React from 'react';
import { shallow } from 'enzyme';

import { Profile } from '../index';

describe('Profile', () => {
  it('should render correctly', () => {
    const component = shallow(
      <Profile />,
    );
    expect(component).toMatchSnapshot();
  });
});
