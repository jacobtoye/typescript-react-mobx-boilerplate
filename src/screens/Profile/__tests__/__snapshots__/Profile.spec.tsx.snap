// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile should render correctly 1`] = `
<Component>
  <Component
    text="Look a profile screen o.O"
  />
</Component>
`;
